rhscl-postgresql-apb/roles/rhscl-postgresql-apb-openshift/README

```diff
@@ -0,0 +1,41 @@
+Postgresql-Demo-APB-Openshift
```

Comment:
hmm, with the env-file the file would have to exist on the container though, so you could only do that if you mounted the file into the container, right?